### PR TITLE
Postpayment change ownershare

### PR DESCRIPTION
**Pull request by vinil-transerve**

### CODE REVIEW

**Review by rajiv-transerve — APPROVED** [review]

File: estate-services/src/main/java/org/egov/ps/service/ApplicationService.java [hunk 1]
```diff
@@ -172,6 +196,101 @@ public void updatePostPayment(ApplicationRequest applicationRequest, Map<String,
...
+//				Property property = propertyRepository.findPropertyById(application.getProperty().getId());
+
+				PropertyCriteria propertySearchCriteria = PropertyCriteria.builder()
+						.propertyId(application.getProperty().getId()).build();
+				List<Property> properties = propertyRepository.getProperties(propertySearchCriteria);
+
+//				properties.forEach(property -> {
+
+				Property property = properties.get(0);
+				System.out.println("properties printed");
```
Comment (rajiv-transerve) [inline estate-services/src/main/java/org/egov/ps/service/ApplicationService.java]: Why do we have a sysout?

File: estate-services/src/main/java/org/egov/ps/service/ApplicationService.java [hunk 2]
```diff
@@ -172,6 +196,101 @@ public void updatePostPayment(ApplicationRequest applicationRequest, Map<String,
...
+				Owner newOwner = new Owner();
+//				Property property = propertyRepository.findPropertyById(application.getProperty().getId());
+
+				PropertyCriteria propertySearchCriteria = PropertyCriteria.builder()
+						.propertyId(application.getProperty().getId()).build();
+				List<Property> properties = propertyRepository.getProperties(propertySearchCriteria);
+
+//				properties.forEach(property -> {
+
+				Property property = properties.get(0);
```
Comment (rajiv-transerve) [inline estate-services/src/main/java/org/egov/ps/service/ApplicationService.java]: why did we replace forEach with properties.get(0). What do we do if there are no properties that are returned ?

**Review by rajiv-transerve — CHANGES_REQUESTED** [review]

File: estate-services/src/main/java/org/egov/ps/service/ApplicationService.java (hunk 1 above)
Comment (vinil-transerve) [reply estate-services/src/main/java/org/egov/ps/service/ApplicationService.java]: I just added during testing, I am removing it

File: estate-services/src/main/java/org/egov/ps/service/ApplicationService.java (hunk 2 above)
Comment (vinil-transerve) [reply estate-services/src/main/java/org/egov/ps/service/ApplicationService.java]: resolved

**Review by rajiv-transerve — APPROVED** [review]

**Pull request merged**